main.go

```diff
@@ -42,12 +42,12 @@ func main() {
 	flag.Parse()
 
 	// Check whether we need to skip empty sources when fetching them or not.
-	skipEmptySources := strings.ToLower(os.Getenv("SKIP_EMPTY_SOURCES")) != "true"
+	skipEmptySources := strings.ToLower(os.Getenv("SKIP_EMPTY_SOURCES")) == "true"
 
 	if psk == "" {
 		log.Fatalf("Need PSK to run availability checks.")
 	}
-	log.Printf("Checking sources with [%v] status from [%v]", *status, host)
+	log.Printf("[host: %s][status: %s][skip_empty_sources: %t] Checking sources", host, *status, skipEmptySources)
```

Comment:
I thought `%s` only worked with limited types and boolean wasn't one of them. Have you checked this log entry looks good?

Reply:
Yes, the log entry looks good. I think they're inline, no? :thinking: 

| Variable | Type | Format type |
|---|---|---|
| `host` | `string` | `%s` |
| `status` | `*string` | `%s` |
| `skipEmptySources` | `boolean` | `%t` |

Reply:
My bad, I read the code wrong!